jenkins_pipelines/environments/manager-5.0-qe-build-validation-NUE

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
+    def minionList = 'sle12sp4_minion, sle12sp4_ssh_minion, ' +
+            'sle12sp5_minion, sle12sp5_ssh_minion, ' +
+            'sle15sp1_minion, sle15sp1_ssh_minion, ' +
+            'sle15sp2_minion, sle15sp2_ssh_minion, ' +
+            'sle15sp3_minion, sle15sp3_ssh_minion, ' +
+            'sle15sp4_minion, sle15sp4_ssh_minion, ' +
+            'sle15sp5_minion, sle15sp5_ssh_minion, ' +
+            'alma9_minion, alma9_ssh_minion, ' +
+            'centos7_minion, centos7_ssh_minion, ' +
+            'liberty9_minion, liberty9_ssh_minion, ' +
+            'oracle9_minion, oracle9_ssh_minion, ' +
+            'rocky8_minion, rocky8_ssh_minion, rocky9_minion, rocky9_ssh_minion, ' +
+            'ubuntu2004_minion, ubuntu2004_ssh_minion, ubuntu2204_minion, ubuntu2204_ssh_minion, ' +
+            'debian11_minion, debian11_ssh_minion, debian12_minion, debian12_ssh_minion, ' +
+            'opensuse154arm_minion, opensuse154arm_ssh_minion, ' +
+            'opensuse155arm_minion, opensuse155arm_ssh_minion, ' +
```

Comment:
add s390 minion and ssh_minion

Reply:
Maybe we should add s390 in an other PR so we can address all the BV pipelines ?

Reply:
AFAICS, Eric already added them to the other BV pipelines.